userguide/config.toml

```diff
@@ -80,22 +80,6 @@ description = "Docsy does docs"
 languageName ="English"
 # Weight used for sorting.
 weight = 1
-[languages.no]
```

Comment:
Did you mean to remove our Norwegian language settings?

Reply:
Yes. Norwegian in the userguide was introduced by me for testing the multilingual taxonomy support. Only the Docsy escape site had the Norwegian content before, I think (and hope).

Reply:
You are right - it was just the example site! (too many sites, I can't remember what features are in what site...)